codebase.md

```diff
@@ -5,4 +5,5 @@ group: Codebase
 
 ---
 
-* (Hydra reference implementation - v0.1)[https://github.com/justincpresley/ndn-hydra]
+
+* [Hydra reference implementation - v0.1](https://github.com/justincpresley/ndn-hydra)
```

Comment:
@justincpresley Can we transfer this repo under hydra-repo (you will still have the same permissions)?